sirepo/pkcli/admin.py

```diff
@@ -44,6 +50,25 @@ def audit_proprietary_lib_files(*uid):
 
 def create_examples():
     """Adds missing app examples to all users"""
+
+    def _create():
+        for sim_type in feature_config.cfg().sim_types:
+            names = [
+                x.name
+                for x in simulation_db.iterate_simulation_datafiles(
+                    sim_type,
+                    simulation_db.process_simulation_list,
+                    {"simulation.isExample": True},
+                )
+            ]
+            for example in simulation_db.examples(sim_type):
+                if example.models.simulation.name not in names:
+                    _create_example(example)
+
+    _access_sim_db_and_callback(_create)
+
+
+def _access_sim_db_and_callback(callback):
```

Comment:
I would call this `_iterate_all_users` and make it a generator so you would use it:

```py
for _ in _iterate_all_users():
     the code that does the work
```